mParticle-Apple-Media-SDKTests/mParticle_Apple_MediaTests.swift

```diff
@@ -531,4 +531,34 @@ class mParticle_Apple_MediaTests: XCTestCase, MPListenerProtocol {
         XCTAssertEqual(mediaEvent1?.streamType, .onDemand)
         XCTAssertTrue(mediaEvent1?.customAttributes == nil)
     }
+    
+    func testMediaTimeSpentWhenLogMediaContentEndCalled() {
```

Comment:
Mind adding a test like this for logPause?

Reply:
@BrandonStalnaker - just added it